main.py

```diff
@@ -148,5 +149,76 @@ async def keen(context):
     await context.send(result)
 
 
-bot.run(TOKEN)
+@bot.command(name='daily', help='Will give the daily leaderboard for specified day')
+async def daily(context, day : str = str(time.localtime(time.time())[7] - 335)):
+    # Only respond if used in a channel called 'advent-of-code'
+    if context.channel.name != 'advent-of-code':
+        return
+    
```

Comment:
Could you add a quick debug print something along the lines of "Daily leaderboard requested for day: <the day>"?

Reply:
Done